EventAggregator.Net.SampleUsage/Samples/AsyncSample.cs

```diff
@@ -4,14 +4,15 @@
 
 namespace EventAggregatorNet.SampleUsage.Samples
 {
+#if ASYNC
     public class AsyncSample
     {
         public static void Run()
         {
             var config = new EventAggregator.Config
             {
                 // Make the marshaler run in the background thread
-                DefaultThreadMarshaler = action => Task.Factory.StartNew(action),
+                DefaultThreadAsyncMarshaler = async action => await Task.Factory.StartNew(action),

```

Comment:
You don't need the async/await pair. Task.Factory.StartNew will never throw (and the only difference async/await will make is that Run() will end up in the call stack of an exception)